### DISCUSSION TITLE
Ability To Add Templated and Also Custom Notes Information Associated With An Image - Print Sizes Available (etc)

### DISCUSSION THREAD

**Discussion by vepicurean** [LycheeOrg/Lychee#2305] [Ideas] [answered]
### Enhancement

Maybe I'm missing if Lychee out of the box has the ability to add templated or custom user information for an uploaded image or album. I've looked, and just can seem to see where this functionality would reside and how it would look. 

I would like to ability to include user information on available print sizes for an image or album ( etc 12in x 18in, 36in x 42in, that I can define the sizes available). Edition information by print size. 12in x 18in - edition of 9, 36in x 42in - edition of 6

**Comment by ildyria** [comment]
I am not sure what you mean. Pictures are in pixel.
You can print them on whatever size you want, it all depends of the DPI selected but that is completely out of the scope of Lychee.

**Reply by vepicurean** [reply]
I'm trying to communicate on the views of an individual image what prints sizes I provide.  I don't see how to communicate that information on an individual image within the existing functionality of the framework.

**Reply by ildyria** [reply]
and you would that in an automated way, basically conversion from pixel to dpi?
Or you can't use the description?

**Reply by vepicurean** [reply]
Nothing that complex. I just want to add custom tags (for lack of a better terminology) to an image in the gallery for the print sizes for each size the edition #1 and how many in the edition. This of the images in the gallery as negatives or slides (again for lack of a better term). 

At the bottom or some other linked screen with the sizes available for the print sizes and edition information. 

Image/Print Name:  Somename #1

Size 12" x 24"
Total in th edition: 6 
Available: 3 - Edition 2 of 6, 3 of 6, 6 of 6

Size: 36" x 42"
Total in the edition: 9
Available: 1 - Edition 9 of 9

Size: 48" x 52"
Total in the edition: 3
Available: 0 - none (sold out)

**Reply by ildyria** [reply]
If you can give me some drawings of what you expect it would be easier for me to understand. This is still quite vague at the moment.

**Reply by vepicurean** [reply]
This example below is more than we need, but it gets the point across. 

https://artandcollectors.com/pages/understanding-your-print-and-the-significance-of-its-edition

When you click on the title at the bottom of a print ( image they call it). It expands to a details page. 

https://artandcollectors.com/products/blackman-charles-visage-ii-6080?_pos=3&_sid=430252a79&_ss=r

In the details you see Edition = 28 ( 28 prints in the edition)

We only need the following information

Print Title
Artist name
Size info -  Xin by Yin  ( for how many different sizes there are)
Edition info - Available which is either a number of how many of the edition for that respective size are available, or sold out

That's all we need. None of the other stuff from the example. We are just using this as a gallery/catalog. We don't sell from this site that we'd use Lychee for.  It is for visually promoting the catalog to clients securely from our in-house Linux server. 

Hope this helps.

**Reply by ildyria** [reply]
Okay I see what you want, I do agree that this could be an interesting use case, but to make this a clean implementation this will require quite a bit of more work.
Also what you want is not Custom Notes, but more like an Edition management.
What you just requested could be easily added manually in the description section of the picture.

**Reply by ildyria** [reply]
That being said, I am sorry to tell you that this will be low priority as there are quite a few more important features to add before. :(

**Reply by vepicurean** [reply]
Understand, I'm happy to help with the UX and testing of the development. Thank you.

**Reply by ildyria** [reply, marked as answer]
Implemented #4388
